google/ads/google_ads/client.py

```diff
@@ -38,16 +38,22 @@
 _DEFAULT_TOKEN_URI = 'https://accounts.google.com/o/oauth2/token'
 _VALID_API_VERSIONS = ['v1', 'v0']
 _DEFAULT_VERSION = _VALID_API_VERSIONS[0]
+_ENV_PREFIX = 'GOOGLE_ADS_'
+_KEYS_ENV_VARIABLES_MAP = {
+    key: _ENV_PREFIX + key.upper() for key in
+    list(_REQUIRED_KEYS) + ['login_customer_id', 'endpoint', 'logging']
+}
 
 class GoogleAdsClient(object):
     """Google Ads client used to configure settings and fetch services."""
 
     @classmethod
-    def _get_client_kwargs_from_yaml(cls, yaml_str):
-        """Utility function used to load client kwargs from YAML string.
+    def _get_client_kwargs(cls, config_data, error_message):
+        """Utility function used to load client kwargs from configuration data
+        dictionary.
 
         Args:
-            yaml_str: a str containing client configuration in YAML format.
+            config_data: a dict containing client configuration.
```

Comment:
Can you add a comment for `error_message`

Reply:
Fixed